plugins/header_rewrite/operator.h

```diff
@@ -55,17 +56,22 @@ class Operator : public Statement
   OperModifiers get_oper_modifiers() const;
   void          initialize(Parser &p) override;
 
-  void
+  // Returns number of executed operators that need to defer call to TSHttpTxnReenable().  It is a fatal error if this
+  // returns more than 1.
```

Comment:
Maybe add a comment specifying that if another rule gets created that needs to be deferred, issue #11549 should be revisited.